Platforms/QemuQ35Pkg/Library/Q35ConfigDataLib/Q35ConfigDataLib.inf

```diff
@@ -0,0 +1,36 @@
+## @file Q35ConfigDataLib.inf
+#  Library instance to carry autogenerated config data.
+#
+#  Copyright (c) Microsoft Corporation.
+#  SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+##
+
+
+[Defines]
+  INF_VERSION         = 0x00010017
+  BASE_NAME           = Q35ConfigDataLib
+  FILE_GUID           = 25D1A00F-E85F-4F1D-8251-02F3E4EDAB1C
+  VERSION_STRING      = 1.0
+  MODULE_TYPE         = BASE
+  LIBRARY_CLASS       = PlatformConfigDataLib

```

Comment:
```suggestion
  LIBRARY_CLASS       = NULL
```
I think this is more of a "NULL" library class?

Reply:
Yep, good point.